### PR TITLE
[Merged by Bors] - chore: CI compiles 'graph'

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: lake-manifest.json [hunk 1]
```diff
@@ -1,37 +1,37 @@
-{"version": 5,
+{"version": 6,
  "packagesDir": "lake-packages",
```
Comment (alexjbest) [inline lake-manifest.json:2]: Is modifying the manifest here intentional?

Comment (semorrison) [reply lake-manifest.json]: No. :-( I've reverted.

**Comment by alexjbest** [comment]
maintainer merge

**Comment by ocfnash** [comment]
bors merge

**Pull request closed without merging**